ksqldb-engine/src/test/java/io/confluent/ksql/planner/plan/JoinNodeTest.java

```diff
@@ -231,6 +232,7 @@ public void setUp() {
 
   @Test
   public void shouldBuildSourceNode() {
+    when(ksqlConfig.getString(KsqlConfig.KSQL_DEFAULT_KEY_FORMAT_CONFIG)).thenReturn("abc");
```

Comment:
It's not great that this had to be added into individuals tests. I think it's preferable to simply disable the rowpartition/rowoffset feature flag in the `KsqlConfig()` constructor. Can you pick up this change in your next PR? I'm going to merge this one now to unblock subsequent work.